cs/__init__.py

```diff
@@ -0,0 +1,143 @@
+def cs_types_encode(key):
+    try:
+        cs_type = _cs_types_encode[key]
+    except KeyError:
+        cs_type = _cs_types_common[key]
+    if cs_type == "NA":
+        raise NotImplementedError("Missing type Mapping")
+    return cs_type
+
+
+def cs_types_decode(key):
+    try:
+        cs_type = _cs_types_decode[key]
+    except KeyError:
+        cs_type = _cs_types_common[key]
+    if cs_type == "NA":
+        raise NotImplementedError("Missing type Mapping")
+    return cs_type
+
+
+def cs_escape_keyword(value):
+    keywords = ["base"]
```

Comment:
We can move this dictionary of escaped keywords outside to eliminate dictionary creation on each call

Reply:
This keyword list is also not complete, we may find a better way to escape.

Reply:
I updated the full reserved words and moved to up

Reply:
We may also convert keyword list to set since it contains more than 100 items now